nss-day-cohort-60/rare-client-tenacious-tators
# Lh add tags

Closes Ticket #11


# posttags.js added to managers and checkbox for tags added to the NewForm.js

---


# Steps to Test

1. Fetch the lh-add-tags branch and switch to it
2. npm start
3. Navigate to posts & click Add Post 
4. Fill out the form: https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSp-eHz2svNqXTmeZeHKpcxcFauhtIYYMuzjQ7xMa4Aa68if0XZ-EclIrt7MZ034F1oI6k&usqp=CAU and choose 1 checkbox
5. When you click publish, verify that the browser navigates you to posts and the new article has updated with title, author, date, category, and tag.
